src/factories/schema-registry-factory.js

```diff
@@ -33,6 +33,15 @@ var SchemaRegistryFactory = function ($rootScope, $http, $location, $q, $log, Ut
     return deferred.promise;
   }
 
+  function schemaParser(schema){
```

Comment:
Change this to `safeJSONParser`. And change the input variable name accordingly.